WooCommerce/Classes/ViewModels/Order Details/OrderDetailsPaymentAlerts.swift

```diff
@@ -0,0 +1,56 @@
+import UIKit
+import WordPressUI
+
+/// A layer of indirection between OrderDetailsViewController and the modal alerts
+/// presented to provide user-facing feedback about the progress
+/// of the payment collection process
+/// It is using a FancyAlertViewController at the moment, but this is the class
+/// to rewrite whenever we have the UI finalized.
+/// https://github.com/woocommerce/woocommerce-ios/issues/3980
+final class OrderDetailsPaymentAlerts {
+    private var alertController: FancyAlertViewController?
```

Comment:
FancyAlertViewController works well, but it does not provide the layout we need. So we need to write a "new" fancy alert (or modify the existing one whatever is easier) to support the layout we need. 

That would happen in #3980 

In the meantime, we can get away with this,